.github/workflows/terraform-apply.yaml

```diff
@@ -21,7 +21,9 @@ jobs:
         fail-fast: false
         matrix:
           repository:
-            - soleng-tf-test-repo
+            - charmed-openstack-upgrader_main
+            # The snapcraft channel for 3.x is not ready
+            # - charmed-openstack-upgrader_3.x
```

Comment:
(non-blocking) I think we should simply remove it

Reply:
Deleted